server.js

```diff
@@ -11,15 +10,8 @@ app.use(function(req, res, next) {
   next();
 });
 
-// parse application/json
-app.use(bodyParser.json())
-
-app.post("/fib", function (request, response) {
-  const num = request.body.number;
-  const result = fibUtils.fibonacci(num)
-  response.send(result)
+app.get("/:number", function (request, response) {
+  response.send( fibUtils.fibonacci( request.params.number ) )
 });
 
-app.get('/', (req, res) => res.send('Hello World! Post to /fib with {"number":10} for a fibonacci sequence'))
```

Comment:
do we need to remove this?  I think it's likely that people go to / endpoint, and then aren't sure what to do.  Can you just provide them with instructions for going to /fib with the number?

Reply:
done!